src/components/post/Post.js

```diff
@@ -198,12 +198,12 @@ const Post = (props) => {
                   <p>
                     {liked ? (
                       <i
-                        className="fas fa-thumbs-up"
+                        class="fas fa-chevron-up"
```

Comment:
shouldn't these stay as className since it's a react component?

Reply:
if I kept className then it wouldn't show the up arrow

Reply:
so after I check it again .I  can change it to classsName, honestly don't know why first time I tried it didn't work, you're right